OpenRA.Mods.Common/Activities/Air/FlyIdle.cs

```diff
@@ -48,10 +48,12 @@ public override bool Tick(Actor self)
 				foreach (var tickIdle in tickIdles)
 					tickIdle.TickIdle(self);
 
-			if (!aircraft.Info.CanHover)
+			if (aircraft.Info.IdleSpeed > 0 || (!aircraft.Info.CanHover && aircraft.Info.IdleSpeed < 0))
```

Comment:
So if I read this correctly we will never fly idle when the idle speed is exactly zero? Shouldn't we mention that in the `Desc`? Or update this to use the normal speed when `IdleSpeed <= 0`?

Reply:
clarified the Description.